modules/enumeratum/build.sbt

```diff
@@ -1,9 +1,17 @@
 import Dependencies.Version._
+import Utilities.forScalaVersions
 
 name := "pureconfig-enumeratum"
 
-crossScalaVersions := Seq(scala212, scala213)
+crossScalaVersions := Seq(scala212, scala213, scala3)
 
 libraryDependencies ++= Seq("com.beachape" %% "enumeratum" % "1.7.2")
 
 developers := List(Developer("aeons", "Bjørn Madsen", "bm@aeons.dk", url("https://github.com/aeons")))
+
+lazy val extraScalacOptions = forScalaVersions {
+  case (3, _) => List("-Yretain-trees")
```

Comment:
Why do we need this here? If it's is really needed, can you please add a comment with an explanation for it?

Reply:
Without it the compiler complains that one should enable "-Yretain-trees". I will add a comment linking to the enumeratum documentation.